repos/re-theme/container/values.yml

```diff
@@ -1,3 +1,13 @@
+actions:
+  tap:
+    build:
+      cmds:
+        - yarn sb:build
+        - rm -rf {{ envs.DOC_BUILD_PATH }}
+        - cp -R {{ envs.DOC_APP_PATH }}/docs {{ envs.DOC_BUILD_PATH }}
```

Comment:
Hmm I must be missing something: isn't this action building and then immediately deleting the build? Then it looks like it's just copying over the docs folder into the build path? Can you untangle this for me?

Reply:
It could be that the ENVs are causing the confusion.
Look at it like this => 
* `DOC_APP_PATH/docs === /keg/tap/docs`
* `DOC_BUILD_PATH === /keg/tap-build`

Given that, if we follow the list of cmds =>
* `yarn sb:build` - creates a new build at `/keg/tap/docs`
  * Builds the application code in it's current state
* `rm -rf {{ envs.DOC_BUILD_PATH }}` - is the same as `rm -rf /keg/tap-build`
  * It removes the **OLD** build it one already exists
  * This is done to make sure we are starting fresh
  * This cmd could come first if we wanted it to
* `cp -R {{ envs.DOC_APP_PATH }}/docs {{ envs.DOC_BUILD_PATH }}` - is the same as `cp -R /keg/tap/docs /keg/tap-build`
  * It copies the **NEW** build created in the first command to the path we just deleted above

So basically it dose this
1. Create a new build of the current tap code
2. Remove the old build code by deleting the previously built folder ( if it exists )
3. Copy the new build to the same location as the old build we just deleted